.github/workflows/ci.yml

```diff
@@ -241,6 +241,10 @@ jobs:
           composer require --dev --with-all-dependencies phpunit/phpunit:^12.1.5
           php php-cs-fixer fix --quiet --rules=php_unit_attributes
 
+      - name: Check configuration for PHPUnit
+        if: matrix.php-version != '7.4' && matrix.php-version != '8.0'
```

Comment:
why such condition?

Reply:
Nah, I am not a fan of commenting every `if` condition, remove it and see CI failing.

Reply:
i'm not a fan of checking every single line with "what will fail on CI if removed", especially as author knows it and everyone else would need to check manually.

writing the actual reason would take as much time as writing "i'm not gonna tell you, check yourself"